lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -1000,278 +997,6 @@ def show_invocation_report_pdf(self, trans: GalaxyWebTransaction, invocation_id,
-                    param, value, step = k, v, inv_step.workflow_step.order_index
-                    parametric_domain.append({"param": str(param), "value": str(value), "step": str(step)})
-            except Exception:
-                continue
-
-        execution_domain = {
-            "script": [{"uri": {"uri": url_for("workflows", encoded_workflow_id=encoded_workflow_id, qualified=True)}}],
-            "script_driver": "Galaxy",
-            "software_prerequisites": software_prerequisites,
-            "external_data_endpoints": [{"name": "Access to Galaxy", "url": url_for("/", qualified=True)}],
-            "environment_variables": kwd.get("environment_variables", {}),
-        }
-
-        extension = [
-            {
-                "extension_schema": "https://raw.githubusercontent.com/biocompute-objects/extension_domain/1.2.0/galaxy/galaxy_extension.json",
-                "galaxy_extension": {
-                    "galaxy_url": url_for("/", qualified=True),
-                    "galaxy_version": VERSION,
-                    # TODO:
-                    # Extend this definition to include more information that is significan for a galaxy workflow/invocation?
-                    # 'aws_estimate': aws_estimate,
-                    # 'job_metrics': metrics
-                },
-            }
-        ]
-
-        error_domain = {
-            "empirical_error": kwd.get("empirical_error", {}),
-            "algorithmic_error": kwd.get("algorithmic_error", {}),
-        }
-
-        bco_dict = {
-            "provenance_domain": provenance_domain,
-            "usability_domain": usability_domain,
-            "extension_domain": extension,
-            "description_domain": {
-                "keywords": keywords,
-                "xref": kwd.get("xref", []),
-                "platform": ["Galaxy"],
-                "pipeline_steps": pipeline_steps,
-            },
-            "execution_domain": execution_domain,
-            "parametric_domain": parametric_domain,
-            "io_domain": {
-                "input_subdomain": input_subdomain,
-                "output_subdomain": output_subdomain,
-            },
-            "error_domain": error_domain,
-        }
-        # Generate etag from the BCO excluding object_id and spec_version, as
-        # specified in https://opensource.ieee.org/2791-object/ieee-2791-schema/-/blob/master/2791object.json
-        etag = hashlib.sha256(json.dumps(bco_dict, sort_keys=True).encode()).hexdigest()
-        bco_dict.update(
-            {
-                "object_id": url_for(
-                    controller=f"api/invocations/{invocation_id}", action="biocompute", qualified=True
-                ),
-                "spec_version": spec_version,
-                "etag": etag,
-            }
-        )
-        return bco_dict
-
-    @expose_api
-    def export_invocation_bco(self, trans: GalaxyWebTransaction, invocation_id, **kwd):
```

Comment:
Is it necessary to remove the API endpoints? Are we sure they are not used?
They are also exposed via BioBlend.

Reply:
I think that was the whole point of the refactoring, unify the invocation export for all formats, but I see your point.
We can mark the old endpoints as deprecated for a while and point to the new endpoints in the docstring.

I'll see if I can reuse as much as possible from the new export system in the old endpoints to keep them minimal.

Reply:
>

        The BioCompute Object endpoints are in beta - important details such
        as how inputs and outputs are represented, how the workflow is encoded,
        and how author and version information is encoded, and how URLs are
        generated will very likely change in important ways over time.

We marked this as beta very explicitly and very intentionally. I would prefer to not maintain the endpoints or put @davelopez time on it but I'm not going to prevent anyone from doing that.